lib/vagrant-service-manager/plugin_util.rb

```diff
@@ -0,0 +1,18 @@
+module Vagrant
+  module ServiceManager
+    module PluginUtil
+      def self.copy_certs_to_host(machine, path)
+        Dir.mkdir(path) unless Dir.exist?(path)
+
```

Comment:
We need to put the message `"# Copying TLS certificates to #{secrets_path}"` here.
This will serve two purposes:
- Display message during box boot up process
- Display message during `env docker`  call given the certs at host are invalid and we needed to copy